CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.12.1-rc1
+
+### Fixed
+
+- Keyboard input not working on macOS with some IMEs like Fig.io
```

Comment:
Why not include the other two fix entries too?

To be honest, we could probably just release master. There haven't been any significant changes.

Reply:
Could you provide hashes? We changed bindings and bumped msrv, so it's not really an option.

Reply:
Just look at the `Fixed` section in the changelog?